templates/psf/default.html

```diff
@@ -26,6 +27,8 @@
 {% block breadcrumbs %}
     {% load sitetree %}
 
+    {% render_active_banner True %}
```

Comment:
Nit: Did Django complained without passing an argument? I don't have a computer to test at the moment, but we might just use `{% render_active_banner %}` here.

Reply:
with suggestion from jaap3, this isn't necessary. I was just being explicit :)